includes/class-omise-wc-myaccount.php

```diff
@@ -61,31 +61,35 @@ public function register_omise_my_account_scripts() {
 				true
 			);
 
-			wp_enqueue_script(
-				'omise-util',
-				plugins_url( '/assets/javascripts/omise-util.js', dirname( __FILE__ ) ),
-				array( 'omise-js' ),
-				WC_VERSION,
-				true
-			);
-
 			wp_enqueue_script(
 				'omise-myaccount-card-handler',
 				plugins_url( '/assets/javascripts/omise-myaccount-card-handler.js', dirname( __FILE__ ) ),
-				array( 'omise-js' , 'omise-util' ),
+				array( 'omise-js' ),
 				WC_VERSION,
 				true
 			);
 
-			wp_localize_script(
-				'omise-myaccount-card-handler',
-				'omise_params',
-				array(
-					'key'             => Omise()->settings()->public_key(),
-					'ajax_url'        => admin_url( 'admin-ajax.php' ),
-					'ajax_loader_url' => plugins_url( '/assets/images/ajax-loader@2x.gif', dirname( __FILE__ ) )
+			$omise_params = array(
+				'key'                            => Omise()->settings()->public_key(),
+				'ajax_url'                       => admin_url( 'admin-ajax.php' ),
+				'ajax_loader_url'                => plugins_url( '/assets/images/ajax-loader@2x.gif', dirname( __FILE__ ) ),
+				'required_card_name'             => __( 'Card Holder\'s name is a required field', 'omise' ),
+				'required_card_number'           => __( 'Card Number is a required field', 'omise' ),
+				'required_card_expiration_month' => __( 'Card Expiry month is a required field', 'omise' ),
+				'required_card_expiration_year'  => __( 'Card Expiry year is a required field', 'omise' ),
+				'required_card_security_code'    => __( 'Card Security code is a required field', 'omise' ),
+				'cannot_create_card'             => __( 'Unable to add a new card.', 'omise' ),
+				'cannot_connect_api'             => __( 'Currently, the payment provider server is under maintain.', 'omise' ),
+				'cannot_load_omisejs'            => __( 'Cannot connect to the payment provider.', 'omise' ),
+				'check_internet_connection'      => __( 'Please make sure that your internet connection is stable.', 'omise' ),
+				'retry_or_contact_support'       => wp_kses(
+					__( 'This incident could happen either from using an invalid card or the payment provider server is under maintainence.<br/>
+					    You may retry again in a couple seconds, or contact our support team if you have any question.', 'omise' ),
+					array( 'br' => array() )
```

Comment:
**tl;dr**
Some of these strings are being used in 2 places. This could raise a potential case of unmatching translation.
However, I'm planning to have a dedicated PR to properly handle this topic as this might be a bit out of scope of this PR (and don't want to make this PR too big).

---
Note, some of these strings are being used in 2 places, at `Omise_Payment_Creditcard` and `Omise_MyAccount`.
This is because we're removing `omise-util.js` and move these invalidation messages back to server side so we can localize it. 

This could raise a potential case where one place gets updated over another causing an unmatching translation.

I'm thinking of housing all the translation strings in one class (i.e. Omise_Localization) so we will have only one place to look for when these needed to be updated.

However, this might be too much for this PR.
I'm planning having a dedicated PR for this topic right after this PR (or, before releasing Konbini payment, at late – this PR may be released along with a PR that was solving 3-D Secure setting feature before I finish the Omise_Localization one).